x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -313,5 +318,6 @@ export interface ActionListApiResponse {
   userIds: string[] | undefined; // users that requested the actions
   commands: string[] | undefined; // type of actions
   data: ActionDetails[];
```

Comment:
In my pending PR, I already changed the `data` property's type of this API (above) to ensure it is accurate of its return value. `outputs` is not returned on the list - [see here](https://github.com/elastic/kibana/pull/139245/files#diff-f7586e6a6b97a380fd6848d32994e7b5e60a6bf1c579bb6b6efeea2df455597dR333))

The same needs to be done for the `hosts` for this API response. Something like:


```typescript
Omit<ActionDetails, 'hosts'> & Pick<Partial<ActionDetails>, 'hosts'>
```